app/views/account/news/_news.html.slim

```diff
@@ -0,0 +1,11 @@
+.news-item
+  .news-form
+    span.badge.badge-warning.float-right.mb-4 = t('.status') if news.is_visible?
```

Comment:
what if news.is_visible? will return false, span element will be crated without text? if yes, better not to create empty html elements on page, the same in line 4.

Reply:
This function was created for admins and leaders of osbb to create post only visible for creators, not simple user or memebers. With this feature, creator can publish post for later editing, or to look how it will be seen after creation. By default, all posts will be visible for users, but if you check off this feature, this post will be visible only for administrators. Screenshot: 
![image](https://user-images.githubusercontent.com/45712870/98680443-17ef5b00-236a-11eb-9793-ddf99975e9f6.png)

Reply:
This if only for yellow check on screenshots, all logic was wrote in controller